Clinical-Genomics/cg
# Travis pylint

Automatically runs lint on the changed code (not all code).

Most of this PR is pep8 except:
- requirements-dev.txt
- .travis.yml
- .gitlint.yaml
- commit _made BaseHandler a super class of the other store Handlers_

How to test:
1. go to the conversation tab in this PR on github
2. scroll down to "Checks"
3. Open the travis build details for the push and the merge 
4. view the build logs
Expected outcome:
Both the logs should contain the line: `The command "git reset --soft ${TRAVIS_COMMIT_RANGE%...*} && git lint" exited with 0.`
and the builds should be successful
